tests/verify_heap.cu

```diff
@@ -103,7 +103,7 @@ int main(int argc, char** argv){
 
   if( computeCapabilityMajor < 2 ) {
     std::cerr << "Error: Compute Capability >= 2.0 required. (is ";
-    std::cerr << computeCapabilityMajor << "."<< computeCapabilityMinor << ")" << std::endl;
+    std::cerr << computeCapabilityMajor << "."<< computeCapabilityMinor << ")\n";
```

Comment:
I think the change is good, to be consistent with the rest. Here it looks to not actually change anything as `cerr` always flushes.

Reply:
You are right!